cmd/expiration-mailer/main.go

```diff
@@ -241,23 +241,24 @@ func (m *mailer) findExpiringCertificates() error {
 		m.log.Info(fmt.Sprintf("expiration-mailer: Searching for certificates that expire between %s and %s and had last nag >%s before expiry",
 			left.UTC(), right.UTC(), expiresIn))
 
-		// If CertStatusOptimizationsMigrated is enabled then we can do this query
-		// in a slightly more efficient way by looking at `cs.NotAfter` instead of
-		// `cert.expires` in the WHERE clause.
-		var certs []core.Certificate
+		// First we do a query on the certificateStatus table to find certificates
+		// nearing expiry meeting our criteria for email notification. We later
+		// sequentially fetch the certificate details. This avoids an expensive
+		// JOIN.
+		var statuses []core.CertificateStatus
```

Comment:
As the only field we fill here is the serial can this be replaced with `[]string`?